crates/templates/src/manager.rs

```diff
@@ -340,9 +345,22 @@ async fn copy_template_into(
         )
     })?;
 
+    write_install_record(dest_dir, source);
+
     load_template_from(id, dest_dir)
 }
 
+fn write_install_record(dest_dir: &Path, source: &TemplateSource) {
+    let layout = TemplateLayout::new(dest_dir);
+    let install_record_path = layout.install_record_file();
+
+    // A failure here shouldn't fail the install
+    let install_record = source.to_install_record();
+    if let Ok(record_text) = toml::to_string_pretty(&install_record) {
+        let _ = std::fs::write(install_record_path, record_text);
```

Comment:
```suggestion
       _ = std::fs::write(install_record_path, record_text);
```

Reply:
WHAT

I DID NOT KNOW YOU COULD DO THIS

THANK YOU BRIAN

Reply:
Can't believe Clippy gets in my face about double references but doesn't bother to tell me about things like this